environment.yml

```
@@ -4,7 +4,7 @@ channels:
   - defaults
 dependencies:
   # Python
-  - python >=3.8,<3.10
+  - python >=3.8
```

```suggestion
  - python >=3.8,<3.11
```

Currently, creating an xcube environment with the one-sided `>=3.8` limit installs Python 3.10, I assume because something in the dependencies is currently requiring `python <3.11`. That dependency will one day suddenly start supporting Python 3.11 and new xcube environments will start getting Python 3.11 by default with no changes on our side. That seems dangerous, so I suggest including the upper limit, but I leave the decision to you.

---

> something in the dependencies is currently requiring `python <3.11`

It would be interesting to know what that something is.

---

> It would be interesting to know what that something is.

Agreed -- I considered investigating, but decided against it since explorations of conda dependency structures can get quite time-consuming.

---

So you know, Python 3.11 is _not_ working?

---

If not, we stay with `python >=3.8` and fix any issues with `3.11` when they occur.

---

I don't know that Python 3.11 is *not* working, but I also don't know how to test that it *will* work. So if you want to take the chance we can leave it without the upper limit :)